onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -169,6 +169,7 @@ Status Gemm<float>::PrePack(const Tensor& tensor, int input_idx,
       prepacked_weights->buffer_sizes_.push_back(packed_b_size);
     }
   }
+
```

Comment:
looks like it's safe to revert every change in this file. (blank lines and `#if 0`s)

Reply:
you are right I used the code for debugging - I will revert back the changes